<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegistrationWindow</class>
 <widget class="QDialog" name="RegistrationWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>466</width>
    <height>207</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_Login_Username">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>58</y>
     <width>62</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Username:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Login_Password">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>98</y>
     <width>62</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Password:</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_login_username">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>58</y>
     <width>122</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_login_password">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>98</y>
     <width>122</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="Line" name="line_between">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>0</y>
     <width>20</width>
     <height>321</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_do_Login">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>168</y>
     <width>191</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>do</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_reg_username">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>52</y>
     <width>151</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_Reg_Username">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>58</y>
     <width>61</width>
     <height>10</height>
    </rect>
   </property>
   <property name="text">
    <string>Username:</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_Withoutusername">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>81</y>
     <width>150</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Without username</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_do_Registration">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>168</y>
     <width>231</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>do</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_reg_password">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>111</y>
     <width>151</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_Reg_Password">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>118</y>
     <width>61</width>
     <height>10</height>
    </rect>
   </property>
   <property name="text">
    <string>Password:</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_withoutpassword">
   <property name="geometry">
    <rect>
     <x>302</x>
     <y>140</y>
     <width>150</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Without password</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_rememberme">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>130</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Remember me</string>
   </property>
  </widget>
  <widget class="Line" name="line_hozi">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>471</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_LogIn">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Log  in</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Registration">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>10</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Registration</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
